### PR TITLE
Feature/all checklists

### PR DESCRIPTION
**Pull request by helenrendich**

### CODE REVIEW

File: backend/src/main/java/com/example/backend/controller/ChecklistController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import java.util.List;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/easy-bikepackr/lists")
+public class ChecklistController {
+
+    private final ChecklistService checklistService;
+
+    @GetMapping("/")
```
Comment (MoinMarcell) [inline backend/src/main/java/com/example/backend/controller/ChecklistController.java:19]:
```suggestion
    @GetMapping
```

Comment (TonyGlimm) [inline backend/src/main/java/com/example/backend/controller/ChecklistController.java:19]: maybe "/all" so it´s more clear that you show every item. But I don't think you have to if you prefer this style

**Review by TonyGlimm — APPROVED** [review]

**Review by HendrikNeuschulz — APPROVED** [review]
LGTM!

File: backend/src/test/java/com/example/backend/controller/ChecklistControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+@AutoConfigureMockMvc
+class ChecklistControllerTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Autowired
+    ChecklistRepository checklistRepository;
+
+    Checklist testChecklist = new Checklist("testId", "testDestination", LocalDate.of(2024, 1, 8));
```
Comment (MoinMarcell) [inline backend/src/test/java/com/example/backend/controller/ChecklistControllerTest.java:29]: This is unused code.

File: backend/src/test/java/com/example/backend/controller/ChecklistControllerTest.java
Comment (MoinMarcell) [inline backend/src/test/java/com/example/backend/controller/ChecklistControllerTest.java:1]: You could test if the list is NOT empty

File: frontend/src/components/App/App.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import ChecklistGallery from "../ChecklistGallery/ChecklistGallery";
+import useChecklistsApi from "../hooks/useChecklistsApi";
+
+
+function App() {
+    const {loading, checklists} = useChecklistsApi()
+    return (
+        <div id="app">
+            <main className="main">
+                {!loading && <ChecklistGallery checklists={checklists}/>}
```
Comment (MoinMarcell) [inline frontend/src/components/App/App.tsx:13]: What if the checklist is loading?

**Review by MoinMarcell — APPROVED** [review]
lgtm so far, you have many files where you have no new line at the end of a file. Some suggestions down below

**Pull request merged**